### PR TITLE
Renaming query plan to execution plan

### PR DESCRIPTION
**Pull request by aasimkhan30**
This PR fixes #

### CODE REVIEW

File: src/sql/workbench/common/editor/query/queryResultsInput.ts [hunk 1]
```diff
@@ -12,15 +12,15 @@ import { QueryPlanState } from 'sql/workbench/common/editor/query/queryPlanState
 import { GridPanelState } from 'sql/workbench/common/editor/query/gridTableState';
 import { QueryModelViewState } from 'sql/workbench/common/editor/query/modelViewState';
 import { URI } from 'vs/base/common/uri';
-import { QueryPlan2State } from 'sql/workbench/common/editor/query/queryPlan2State';
+import { ExecutionPlanState } from 'sql/workbench/common/editor/query/executionPlanState';
 
 export class ResultsViewState {
 	public readonly gridPanelState: GridPanelState = new GridPanelState();
 	public readonly chartState: ChartState = new ChartState();
 	public readonly queryPlanState: QueryPlanState = new QueryPlanState();
```
Comment (Charles-Gagnon) [inline src/sql/workbench/common/editor/query/queryResultsInput.ts:20]: executionPlanState?

File: src/sql/workbench/contrib/executionPlan/browser/widgets/nodeSearchWidget.ts [hunk 2]
```diff
@@ -17,10 +17,10 @@ import { SelectBox } from 'sql/base/browser/ui/selectBox/selectBox';
 import { InputBox } from 'sql/base/browser/ui/inputBox/inputBox';
 import { isString } from 'vs/base/common/types';
 
-const CONTAINS_DISPLAY_STRING = localize("queryPlanSearchTypeContains", 'Contains');
-const EQUALS_DISPLAY_STRING = localize("queryPlanSearchTypeEquals", 'Equals');
+const CONTAINS_DISPLAY_STRING = localize("executionPlanSearchTypeContains", 'Contains');
```
Comment (Charles-Gagnon) [inline src/sql/workbench/contrib/executionPlan/browser/widgets/nodeSearchWidget.ts:20]: If you're updating just the IDs you should go through and update the xlf/lcl files as well so the loc team doesn't have to do a bunch of extra work just to update the IDs

File: src/sql/workbench/services/query/common/queryRunner.ts [hunk 3]
```diff
@@ -73,8 +73,8 @@ export default class QueryRunner extends Disposable {
 	private readonly _onQueryPlanAvailable = this._register(new Emitter<IQueryPlanInfo>());
```
Comment (Charles-Gagnon) [inline src/sql/workbench/services/query/common/queryRunner.ts:73]: _onExecutionPlanAvailable?

**Review by Charles-Gagnon — APPROVED** [review]
You may want to do a second pass. I noticed a couple of "queryPlan" and "Query Plan" strings still hanging around.

File: src/sql/workbench/services/query/common/queryRunner.ts (hunk 3 above)
Comment (aasimkhan30) [reply src/sql/workbench/services/query/common/queryRunner.ts:73]: This belongs to old query plan feature.

**Review by alanrenmsft — APPROVED** [review]

**Review by lewis-sanchez — APPROVED** [review]

File: src/sql/workbench/common/editor/query/queryResultsInput.ts (hunk 1 above)
Comment (aasimkhan30) [reply src/sql/workbench/common/editor/query/queryResultsInput.ts:20]: That belongs to the old query plan feature. That is not completely removed as of now.

**Pull request merged**